composer.json

```diff
@@ -15,13 +15,12 @@
         }
     ],
     "autoload": {
-        "psr-0": {
-            "Mopa\\Bundle\\BootstrapBundle": ""
+        "psr-4": {
+            "Mopa\\Bundle\\BootstrapBundle\\": ""
         }
     },
-    "target-dir": "Mopa/Bundle/BootstrapBundle",
     "require": {
-        "symfony/symfony": "~2.3|~3.0",
+        "symfony/symfony": "^3.3",
```

Comment:
Is this the right limiter? or will this not work with 4 or 5?

Reply:
I don't think the bundle will work with anything greater than 3 atm.  The tests won't run unless it's using 2, but the tests don't run anyways... it'll require a great deal of research on this one to get it working properly with 4 (and tests working).  This is just a fix to get it working properly on mybuilder/mybuilder without throwing deprecations.

Reply:
I wouldn't want to commit to saying it'll work with 4, as I don't really know how it works

Reply:
The repo it's branched from works with Symfony 4 I think.  So might be a simple task.

Reply:
sure, makes sense